### PR TITLE
radio+checkbox: implement `hovered` state (close #657)

### LINKED ISSUE #657: radio+checkbox: implement `hovered` state

### PR DESCRIPTION
**Pull request by dfilatov**
I've moved `_hovered` behaviour to `base-control` on desktop level to reduce copy-paste.
cc @narqo @mishanga @gela-d

### CODE REVIEW

File: desktop.blocks/base-control/base-control.spec.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    });
+
+    describe('hover', function() {
+        it('should have "hovered" mod on mouseover', function() {
+            baseControl.domElem.mouseover();
+            baseControl.hasMod('hovered').should.be.true;
+        });
+
+        it('should delete "hovered" mod on mouseout', function() {
+            baseControl.domElem.mouseout();
```
Comment (sipayRT) [inline desktop.blocks/base-control/base-control.spec.js:25]: before mouseout event you should **set** "hovered" mod

Comment (dfilatov) [inline desktop.blocks/base-control/base-control.spec.js:25]: You're definitely right. Fixed.

File: desktop.blocks/base-control/base-control.spec.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                elem : 'control',
+                tag : 'input',
+                attrs : { name : 'blah' }
+            }
+        },
+        baseControl, control;
+
+    beforeEach(function() {
+        baseControl = buildControl(bemjson).bem('base-control');
+        control = baseControl.elem('control');
```
Comment (narqo) [inline desktop.blocks/base-control/base-control.spec.js]: Guess this one is useless

Comment (dfilatov) [inline desktop.blocks/base-control/base-control.spec.js]: fixed

**Comment by narqo** [comment]
looks fine

**Pull request merged**